include/onnxruntime/core/common/common.h

```diff
@@ -242,4 +242,16 @@ inline std::wstring ToWideString(const std::wstring& s) { return s; }
 inline std::string ToWideString(const std::string& s) { return s; }
 #endif
 
+#pragma region Session_Options_Configs
+
```

Comment:
Keep these config definitions here for now.
When we have more configs defined, we can move them into a separated head file

Reply:
nit: Would prefer a separate header file now. We can put the strings in an appropriate namespace like onnxruntime::config_keys, and it won't break user's builds later when we move it.